components/KitText.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import { Text } from 'react-native';
+import * as Font from 'expo-font';
+
+/*
+props expected
+  children
+  style
+  color
+  fontWeight
+  size
+*/
+export default function KitText(props) {
+  const [fontLoaded, setFontLoaded] = useState("");
+  const defaultFontStyles = {marginTop: "auto", marginBottom: "auto", textAlign: "center"}; //EVAN TODO: move this to constants
+
+  const loadFontAssets = async () => {
+    await Font.loadAsync({
+      // EVAN TODO: move the requires to constants
+      'hind-regular': require('../assets/fonts/Hind/Hind-Regular.ttf'),
```

Comment:
Are we trying to change to the other font?

Reply:
Yep, poligon is in business, but moved the async loading to `App.js` and only loaded the default styles here.